src/gc/gcpriv.h

```diff
@@ -781,6 +779,9 @@ class generation
     int             gen_num;
 
 #ifdef FREE_USAGE_STATS
+    #pragma message ("FREE_USAGE_STATS changes the layout of the 'generation' class, which is known to the DAC. " \
```

Comment:
this wouldn't be defined for shipped ret builds for sure. but this could be heavily used during experimenting/giving out private ret builds. since one of the purposes for the LocalGC project is to make giving out private builds easier it would be really nice to make this work.